openedx/core/djangoapps/user_authn/utils.py

```diff
@@ -76,19 +76,23 @@ def generate_username_suggestions(username):
     """ Generate 3 available username suggestions """
     max_length = USERNAME_MAX_LENGTH
     short_username = username[:max_length - 4] if max_length is not None else username
```

Comment:
Given the new range, does this need to be `- 6` now?

Reply:
Yup, will update that.